Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/) for ChainSafe Files/Chainsafe Files user interface.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/chainsafe-files/-/chainsafe-files-user-interface/horizontal-auto.svg)

Commit:
Translated using Weblate (German)

Currently translated at 90.8% (218 of 240 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/de/

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (248 of 248 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/fr/

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

RyRy79261 commented:
So theres no changes here 😅

FSM1 commented:
> So theres no changes here 😅

I think that this is due to the fork merging all the existing PR's that were merged onto `dev`

Tbaut commented:
yeah that bot isn't super clever :D
I tried to figure out why the other day, but failed at doing so, I'll take a look in the next days again, the settings of Weblate are far from easy to navigate!

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

Commit:
Translated using Weblate (French)

Currently translated at 99.6% (258 of 259 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/fr/

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ChainSafe/files-ui?pullRequest=1243) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ChainSafe/files-ui?pullRequest=1243) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ChainSafe/files-ui?pullRequest=1243) it.</sub>

Commit:
Translated using Weblate (French)

Currently translated at 99.6% (258 of 259 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/fr/

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

Tbaut commented on packages/files-ui/src/locales/fr/messages.po:
```diff
@@ -69,7 +70,7 @@ msgid "Browser:"
 
 msgid "Bucket id"
-msgstr ""
+msgstr "Identifiant du seau"
```
```suggestion
msgstr "Identifiant du Bucket"
```

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

weblate committed:
Merge remote-tracking branch 'origin/dev' into dev

Tbaut committed:
Merge branch 'dev' into weblate-chainsafe-files-chainsafe-files-user-interface
